packages/cms-editor/src/main/control/AddContentObject.tsx

```diff
@@ -0,0 +1,149 @@
+    setOpen(open);
+    if (open) {
+      initializeDialog();
+    }
+  };
+
+  const [name, setName] = useState('');
+  const [namespace, setNamespace] = useState('');
+  const [values, setValues] = useState<MapStringString>({});
+
+  const locales = useMeta('meta/locales', context, []).data;
+  const { clientLanguageTag, languageDisplayName } = useClientLanguage();
+  const isClientLanguageInCms = useMemo(() => locales.includes(clientLanguageTag), [clientLanguageTag, locales]);
+
+  const initializeDialog = () => {
+    setName('NewContentObject');
+    setNamespace(initialNamespace(contentObjects, selectedContentObject));
+    setValues(isClientLanguageInCms ? { [clientLanguageTag]: '' } : {});
+  };
+
+  const changeValue = (languageTag: string, value: string) => {
+    const newValues = structuredClone(values);
+    newValues[languageTag] = value;
+    setValues(newValues);
+  };
+
+  const client = useClient();
+  const queryClient = useQueryClient();
+  const { dataKey } = useQueryKeys();
+  const addContentObject = (event: React.MouseEvent<HTMLButtonElement> | KeyboardEvent) => {
+    const uri = `${namespace}/${name}`;
+    client.create({ context, contentObject: { uri, type: 'STRING', values } });
+    queryClient.invalidateQueries({ queryKey: genQueryKey('data') }).then(() => {
+      const data: CmsData | undefined = queryClient.getQueryData(dataKey({ context, languageTags: [clientLanguageTag] }));
+      const selectedContentObject = data?.data
+        .filter((contentObject: ContentObject) => contentObject.type !== 'FOLDER')
+        .findIndex(co => co.uri === uri);
+      setSelectedContentObject(selectedContentObject);
+      selectRow(table, String(selectedContentObject));
+      setOpen(event.ctrlKey || event.metaKey);
+    });
+  };
+
+  const { addContentObject: shortcut } = useKnownHotkeys();
+  useHotkeys(shortcut.hotkey, () => onOpenChange(true), { scopes: ['global'], keyup: true, enabled: !open });
+  const enter = useHotkeys(['Enter', 'mod+Enter'], e => addContentObject(e), { scopes: ['global'], enabled: open, enableOnFormTags: true });
+
+  return (
+    <Dialog open={open} onOpenChange={onOpenChange}>
+      <TooltipProvider>
+        <Tooltip>
+          <TooltipTrigger asChild>
+            <DialogTrigger asChild>
+              <Button icon={IvyIcons.Plus} aria-label={shortcut.label} />
+            </DialogTrigger>
+          </TooltipTrigger>
+          <TooltipContent>{shortcut.label}</TooltipContent>
+        </Tooltip>
+      </TooltipProvider>
+      <DialogContent onCloseAutoFocus={e => e.preventDefault()}>
+        <DialogHeader>
+          <DialogTitle>{t('dialog.addContentObject.title')}</DialogTitle>
+        </DialogHeader>
+        <DialogDescription>{t('dialog.addContentObject.description')}</DialogDescription>
+        <Flex direction='column' gap={3} ref={enter} tabIndex={-1}>
+          <BasicField label={t('common:label.name')} aria-label={t('common:label.name')}>
```

Comment:
I'm not sure if a aria-label is really needed if you have a normal label. I think not.